### PR TITLE
Jp/api keys crud

### PR DESCRIPTION
**Pull request by juanpardo**
## Summary of the changes / Why this is an improvement
Added support for creating, listing, deleting, activating and deactivating user API keys.

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: tests/commands/test_api_keys.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        RequestMethod.DELETE,
+        f"/api/v2/users/me/api-keys/key-to-be-deleted/",
+        params=None,
+    )
+
+
+@mock.patch.object(Client, "request", return_value=({}, None))
+@pytest.mark.parametrize("target_status", [True, False])
+def test_api_keys_edit(mock_request, target_status):
+    target_status = True
```
Comment (plaharanne) [inline tests/commands/test_api_keys.py:48]: I think you forgot to remove this line.

File: tests/commands/test_api_keys.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    if target_status:
+        target_status_str = "true"
+    else:
+        target_status_str = "false"
```
Comment (plaharanne) [inline tests/commands/test_api_keys.py:52]: just a suggestion: you could also do directly `str(target_status).lower()`

**Review by plaharanne — COMMENTED** [review]
It looks good. I just have 2 small feedback

File: croud/apikeys/commands.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        data=data,
+        errors=errors,
+        success_message="API key deleted.",
+        output_fmt=get_output_format(args),
+    )
+
+
+def api_keys_edit(args: Namespace) -> None:
+    body = {"active": args.active}
+    print_info(f"args.active is: {args.active}")
```
Comment (SStorm) [inline croud/apikeys/commands.py:64]: This should probably not be here?

**Review by SStorm — COMMENTED** [review]
Nice work, just one minor thing to tidy up!

**Comment by juanpardo** [comment]
Thank you both for the feedback!

**Review by SStorm — APPROVED** [review]

**Pull request merged**